hushline/settings.py

```diff
@@ -406,9 +406,10 @@ def update_pgp_key() -> Response | str:
             db.session.commit()
             flash("👍 PGP key updated successfully.")
             return redirect(url_for(".index"))
-        return render_template("settings.html", form=form)
 
-    @bp.route("/update_smtp_settings", methods=["GET", "POST"])
+        return redirect(url_for(".index"))
+
+    @bp.route("/update-smtp-settings", methods=["POST"])
```

Comment:
do we need to update the routes in the `settings.html` template to reflect these changes?

Reply:
after reading the docs, it looks like ~url_for` accepts the name of the function, not the url so we don't need to make any changes to the template.